lib/spack/spack/cmd/install.py

```diff
@@ -82,6 +83,9 @@ def setup_parser(subparser):
     cache_group.add_argument(
         '--no-cache', action='store_false', dest='use_cache', default=True,
         help="do not check for pre-built Spack packages in mirrors")
+    subparser.add_argument(
+        '--unsigned', action='store_true', dest='unsigned', default=False,
```

Comment:
could we call this `--no-check-signature` and make the help string "do not check signatures of binary packages"?

Reply:
Done, left the internal destination variable `unsigned` so that it corresponds to the `buildcache` argument.